src/Mvc/Mvc.Core/src/ControllerBase.cs

```diff
@@ -1870,7 +1870,10 @@ public virtual ConflictObjectResult Conflict([ActionResultObjectValue] ModelStat
                 detail: detail,
                 instance: instance);
 
-            return new ObjectResult(problemDetails);
+            return new ObjectResult(problemDetails)
```

Comment:
@huysentruitw we'll also need to something similar in https://github.com/aspnet/AspNetCore/pull/14672/files#diff-8aa2822b0a420b6f8a04153ca4312519L1949.

Reply:
Ok, I will fix this when back home.

Also:
* Do we leave the `statusCode` field in the response body?
* I have used aspnet:release/3.1-preview1 as target branch, is that ok?